interface-definitions/interfaces_wireless.xml.in

```diff
@@ -599,6 +599,15 @@
               <help>Wireless security settings</help>
             </properties>
             <children>
+              <leafNode name="bssid">
```

Comment:
Is BSSID required for WPA enterprise? Or is it the SSID you try to connect to?

Reply:
If this is required please add it in a separate PR. This way it's easier for us the backport changes to other (e.g. LTS) branches